modules/ROOT/pages/http/http-request-ref.adoc

```diff
@@ -162,7 +162,19 @@ A range of failure status codes is defined by two ASCII `..` full stop character
 
 == Configure a Target
 
-By default, the body of a request is taken from the `#[payload]` of the incoming Mule message and the response is sent onwards as the `#[payload]` of the output Mule message. You can change this default behavior through the General > Request > Body and General > Output > Target Variable attributes. Use this attribute to specify a location other than payload for the output data, such as a variable.
+By default, the body of a request is taken from the `#[payload]` of the incoming Mule message and the response is sent onwards as the `#[payload]` of the output Mule message. You can change this default behavior through the General > Request > Body and General > Output > Target Variable attributes. Use this attribute to specify a location other than payload for the output data, such as a variable. If you set the Target Variable the output Mule message wont be on the flow. For save more data than the payload, you need to modify the target value.
```

Comment:
This expanded explanation is both wrong and incomplete. It also doesn't make sense to explain the target attribute over and over again on each connector. This should instead link to https://docs.mulesoft.com/mule-runtime/4.2/target-variables where `target` and `targetValue` attributes are explained in length.